src/providers/hyperbolic/audio.ts

```
@@ -0,0 +1,190 @@
+    if (config.language) {
+      body.language = config.language;
+    }
+
+    logger.debug(`Calling Hyperbolic Audio API: ${JSON.stringify(body)}`);
+
+    const headers = {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${apiKey}`,
+    } as Record<string, string>;
+
+    let data: any, status: number, statusText: string;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)